### PR TITLE
Update whats-new-in-defender-for-office-365.md

### PR DESCRIPTION
**Pull request by ajaj-shaikh**
Update on BCL now being available in advanced hunting

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ For more information on what's new with other Microsoft Defender security produc
 - This change impacts all users who log in to the Office 365 Security and Compliance portal (protection.office.com), including security teams and end-users who access the Email Quarantine experience, at the **Microsoft Defender Portal** > **Review** > **Quarantine**.
 - Redirection is enabled by default and impacts all users of the Tenant.
 - Global Administrators and Security Administrators can turn on or off redirection in the Microsoft 365 Defender portal by navigating to **Settings** > **Email & collaboration** > **Portal redirection** and switch the redirection toggle.
+- Bulk Complaint Level is now available in EmailEvents table in Advanced Hunting with numeric BCL values from 0 to 9. A higher BCL score indicates that bulk message is more likely to generate complaints and is more likely to be a spam.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md]:
```suggestion
- Bulk complaint level (BCL) is now available in the EmailEvents table in advanced hunting with numeric BCL values ranging from 0 to 9. A higher BCL score indicates that bulk messages are more likely to generate complaints and are more likely to be spam.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ajaj-shaikh, kindly commit the suggested changes. Many thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks

**Comment by chrisda** [comment]
Topic file was renamed, and the proposed change is already present in the September 2022 section, so closing this PR.

**Pull request closed without merging**